packages/strapi-admin/ee/admin/containers/Roles/CreatePage/ContentTypeCollapse/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import Collapse from './Collapse';
+import CollapsePropertyMatrix from './CollapsePropertyMatrix';
+import { getAvailableActions } from './utils';
+import Wrapper from './Wrapper';
+
+const ContentTypeCollapse = ({
+  allActions,
+  contentTypeName,
+  index,
+  isActive,
+  onClickToggleCollapse,
+  properties,
+}) => {
+  const handleClickToggleCollapse = useCallback(() => {
```

Comment:
The only component receiving  `handleClickToggleCollapse ` is not memorized, it will re-render anyways when this component will re-render

Reply:
Fixed in #9405